.eslintrc.cjs

```diff
@@ -1,34 +1,60 @@
+/** @type {import('eslint').Linter.Config} */
 module.exports = {
 	root: true,
 	parser: '@typescript-eslint/parser',
 	extends: [
 		'eslint:recommended',
 		'plugin:@typescript-eslint/recommended',
+		'plugin:svelte/recommended',
 		'plugin:storybook/recommended',
 		'prettier',
 	],
-	plugins: ['svelte3', '@typescript-eslint'],
-	ignorePatterns: ['*.cjs'],
+	plugins: ['@typescript-eslint'],
 	overrides: [
 		{
 			files: ['*.svelte'],
-			processor: 'svelte3/svelte3',
+			parser: 'svelte-eslint-parser',
+			parserOptions: {
+				parser: '@typescript-eslint/parser',
+			},
 		},
 	],
-	settings: {
-		'svelte3/typescript': () => require('typescript'),
-	},
 	parserOptions: {
 		sourceType: 'module',
-		ecmaVersion: 2020,
+		ecmaVersion: 'latest',
+		extraFileExtensions: ['.svelte'],
 	},
 	env: {
 		browser: true,
-		es2017: true,
+		es2024: true,
 		node: true,
 	},
 	rules: {
+		// eslint
 		'no-console': 'warn',
+
+		// @typescript-eslint
+		'@typescript-eslint/no-unused-vars': [
+			'warn',
+			{
+				argsIgnorePattern: '^_',
+				varsIgnorePattern: '^_',
+			},
+		],
+
+		// eslint-plugin-svelte
+		'svelte/no-target-blank': 'error',
```

Comment:
this is the one forcing us to add `noopener`. perhaps we should remove it. since we're not compiling the docs in legacy mode it serves no purpose
```suggestion
```

Reply:
Shouldn't we just update the rule configuration to https://sveltejs.github.io/eslint-plugin-svelte/rules/no-target-blank/#allow-referrer-true here to make the rule require rel="noopener" but not noreferrer?

Reply:
It should be the other way around though, requiring `noreferrer` but not `noopener`.

Reply:
I agree: https://stackoverflow.com/questions/57628890/why-people-use-rel-noopener-noreferrer-instead-of-just-rel-noreferrer

Then that rule option makes no sense, how should we proceed?

Reply:
I posted in `eslint-plugin-svelte` to see if we can address it there: https://github.com/sveltejs/eslint-plugin-svelte/issues/571#issuecomment-1691839668

Reply:
For now I'll merge this PR and we can update the plugin once its fixed